src/data/moves/move-attrs/secret-power-attr.ts

```diff
@@ -131,7 +131,13 @@ export class SecretPowerAttr extends ChanceBasedMoveEffectAttr {
   }
 
   /** Secret Power ignores the move chance bonus from the Water + Fire Pledge combo effect */
-  override getMoveChance(user: Pokemon, target: Pokemon, move: Move, showAbility: boolean = false): number {
+  override getMoveChance(
+    user: Pokemon,
+    target: Pokemon,
+    move: Move,
+    _selfEffect: boolean,
```

Comment:
Should there be a default value of `false` here ?

```suggestion
    _selfEffect: boolean = false,
```